### PR TITLE
fix scroll emit

### PR DESCRIPTION
**Pull request by RomanGavrylenko**
Method emit is incorrect use. Second parameter must be an object. https://www.npmjs.com/package/mitt

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Hi @RomanGavrylenko, thanks for your contribution! We'll have a look at your proposed changes.

Some things you can do first:

- Fix linter errors (cf. [travis build](https://travis-ci.org/bpmn-io/scroll-tabs/builds/578495741?utm_source=github_status&utm_medium=notification)). You can use `npm run lint -- --fix` to auto-fix them
- Have a look at our [contribution guidelines](https://github.com/bpmn-io/bpmn-js/blob/develop/.github/CONTRIBUTING.md#creating-a-pull-request), especially regarding our commit message format. In your case, it would be something like

`chore(scroll-tabs): use object for emitting`

*  [Confirm the CLA](https://github.com/bpmn-io/scroll-tabs/pull/4#issuecomment-526316350)
*  Re-think about a possible test case for your change. Is there any change to test the changes you made in the current test environment?

**Comment by pinussilvestrus** [comment]
I tried out an integration inside the Camunda Modeler. It revealed that this changed would lead to breaking changes since we expect a single node on scroll events, cf. [`TabLinks`](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/primitives/TabLinks.js#L84)

Same thing for `bpmn-js-properties-panel` (and our other properties panels). It reveals we never actually use the second and third param, only the `newActiveNode`.

tldr: I'd not pull in these changes as it would break too much. I'd rather simply remove the second and third argument when emitting the event because they are useless.

**Comment by pinussilvestrus** [comment]
So I think we can close this since it does not offer a proper improvement. @RomanGavrylenko feel free to re-open it when you think we should have a further discussion about that.
